business/definitionService.js

```diff
@@ -498,7 +509,16 @@ class DefinitionService {
   _getDefinitionCoordinates(coordinates) {
     return Object.assign({}, coordinates, { tool: 'definition', toolVersion: 1 })
   }
+
+  _getCacheKey(coordinates) {
+    return (
+      'co_' +
```

Comment:
nit but as a clarity thing, the cache key should relate to the type of the value not the type of the key. So here perhaps `'def_'` rather than `'co_'` as this is the key used to store definitions in the flat cache namespace.